interface/billing/edih_view.php

```diff
@@ -609,411 +669,432 @@ function(data) {
-                    appendTo: '#tbcsvhist',
-                    height: 'auto',
-                    width: 568,
-                    maxWidth: 616,
-                    title: <?php echo xlj("Encounter EDI Record"); ?>,
-                    close: function(event, ui) {
-                        $(this).empty();
-                        $(this).dialog('close');
-                    }
-                };
-        $.ajax({
-            type: "GET",
-            url: $('#formcsvhist').attr('action'),
-            data: $('#formcsvhist').serialize(), //{ csvenctr: chenctr },
-            dataType: "html",
-            success: [ function(data){
-                $('<div/>', {'class':'edihDlg', 'id':'link-'+($(this).index()+1)})
-                    .appendTo('#tbcsvhist').html($.trim(data)).dialog(histopts).dialog('open');
+        /* ****************
+         * begin csv tables section
+         * the csv tables are displayed using jquery dataTables plugin
+         * here, the 'success' action is to execute an array of functions
+         * the helper function bindlinks() applies jquery .on method
+         * so most links will open a jquery-ui dialog
+         */
+        $('#formcsvtables').on('submit', function(e) {
+            e.preventDefault();
+            e.stopPropagation();
+            // verify a csv file is selected
+            if ($('#csvselect').val() == '') {
+                $("#tblshow").html('<?php echo xla("No table selected! Select a table."); ?>');
+                return false;
+            }
+            $.ajax({
+                type: 'get',
+                url: "edih_main.php",
+                data: $('#formcsvtables').serialize(),
+                dataType: "html",
+                success: [
+                    function(data) {
+                        $('#tblshow').html(data);
+                        $('#tblshow').css('maxWidth', 'fit-contents');
+                        $('#tblshow table#csvTable').DataTable({
+                            'processing': true,
+                            'scrollY': '300px',
+                            'scrollCollapse': true,
+                            'scrollX': true,
+                            'paging': true,
+                            <?php // Bring in the translations ?>
+                            <?php require($GLOBALS['srcdir'] . '/js/xl/datatables-net.js.php'); ?>
+                        });
+                    },
+                ]
+            });
+        });
+
+        // csv encounter history
+        $('#formcsvhist').on('submit', function(e) {
+            e.preventDefault();
+            $('#tbcsvhist').empty();
+            var chenctr = $('#histenctr').value;
+            var histopts = {
+                modal: false,
+                appendTo: '#tbcsvhist',
+                height: 'auto',
+                width: 568,
+                maxWidth: 616,
+                title: '<?php echo xlj("Encounter EDI Record"); ?>',
```

Comment:
should not add those quotes (the xlj does this via json_encode())

Reply:
Then let's not use xlj cause my php editor (Brackets) thinks its an error.

Reply:
I am going to pretend you didn't say that :)
(json_encode is the standard that codebase uses for cross-scripting prevention in javascript context)
Try to flip the inside quotes to single quotes:
```
title: <?php echo xlj('Encounter EDI Record'); ?>,
```